### ISSUE TITLE
Crash when passing --profile

### ISSUE THREAD

**Issue by mprobst** [bazelbuild/bazel-watcher#58]
```
$ ibazel test --profile=myprofile.out  test:golden_test
State: QUERY
Querying for BUILD files...
Error running Bazel exit status 2
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x40ec1e3]

goroutine 1 [running]:
main.(*IBazel).queryForSourceFiles(0xc42009a000, 0xc420018240, 0x3f, 0x0, 0xc4200cc000, 0xc4200ce000)
	ibazel/ibazel.go:316 +0x1b3
main.(*IBazel).watchFiles(0xc42009a000, 0xc420018240, 0x3f, 0xc420062060)
	ibazel/ibazel.go:345 +0x5a
main.(*IBazel).iteration(0xc42009a000, 0x413b590, 0x4, 0xc420055e18, 0xc42000c060, 0x2, 0x2, 0xc42001c0f0, 0x28)
	ibazel/ibazel.go:204 +0x91e
main.(*IBazel).loop(0xc42009a000, 0x413b590, 0x4, 0xc420055e18, 0xc42000c060, 0x2, 0x2, 0xc420055e20, 0xc420010170)
	ibazel/ibazel.go:176 +0xf6
main.(*IBazel).Test(0xc42009a000, 0xc42000c060, 0x2, 0x2, 0x2, 0x2)
	ibazel/ibazel.go:168 +0x80
main.handle(0xc42009a000, 0x7fff5fbff7ff, 0x4, 0xc420010160, 0x2, 0x2)
	ibazel/main.go:110 +0x22e
main.main()
	ibazel/main.go:99 +0x1bf
```

**Comment by dougkoch** [comment]
@mprobst, is your ibazel up-to-date? Running your command on the tsickle repo, I was able to reproduce this crash from cd96809, but not from HEAD. It appears that @achew22 fixed this in #48 (his diff on line 313 of ibazel.go, which prevents it from attempting to loop over non-existent build targets when the query fails).

**Comment by mprobst** [comment]
Might have been out of date, I'm not sure if I upgraded before or after.
